markdown/en/ff-searchbox.md

```diff
@@ -1,9 +1,18 @@
 ## Setup the communication
-First, add the `ff-communication` tag to every page that uses the **FACT-Finder Webcomponents** . With this element you define the location of your FACT-Finder Server in the `url` attribute and which `channel` you want to use in the channel attribute.
+First, add the `ff-communication` tag to every page that uses
+**FACT-Finder Webcomponents**. This element is, in part, used to define
+the location of your FACT-Finder Server using the `url` attribute and
+the `channel` you want to use using the `channel` attribute.
```

Comment:
This whole sentence was to long for me. Maybe something like the following is better:

First, add the `ff-communication` tag to every page that uses **FACT-Finder Webcomponents**. The `url` attribute must match the location of your FACT-Finder Server. The `channel` attribute must relate to a channel in your FACT-Finder Server. These two attributes are required and define the target of each request to FACT-Finder.